### DISCUSSION TITLE
Your favorite fruit is ?

### DISCUSSION THREAD

**Discussion by Ngochuy2137** [NAIST-SE/AttractingContributors#1136] [2024 Attraction Contributions]
**_😊 What is your favorite fruit ?_**
Please contribute my repo to share your hobby 😊 [repo_link](https://github.com/Ngochuy2137/AttractingContributors-fruits)
_Ex: Mango, Strawberry, apple, avocado, banana, blackberry, blueberry, cherry, coconut, grape, grapefruit, kiwi, lemon, lime, orange, watermelon, ..._
![image_2024-06-25_032528837](https://github.com/NAIST-SE/AttractingContributors/assets/92006039/35645382-4905-4c22-8a62-bb0a8df5d12e)

**Comment by Drobotsan** [comment]
Hi, I contribute to your project.
Please accept my pull request.

Would you be kind to contribute with my project :D
https://github.com/Drobotsan/Places-in-the-world

Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1140

**Reply by Ngochuy2137** [reply]
Of course my friend, I did it already

**Comment by Marsall-k** [comment]
I have just made a PR on your Repo!
Could you contribute to mine?
https://github.com/Marsall-k/AttractingContributors-Pokemon

**Reply by Ngochuy2137** [reply]
Thank you my friend ! Melon is a good option
